.CI/chatterino-installer.iss

```diff
@@ -0,0 +1,73 @@
+; Script generated by the Inno Setup Script Wizard.
+; SEE THE DOCUMENTATION FOR DETAILS ON CREATING INNO SETUP SCRIPT FILES!
+
+#define MyAppName "Chatterino"
+#define MyAppVersion "2.4.2"
+#define MyAppPublisher "Chatterino Team"
+#define MyAppURL "https://www.chatterino.com"
+#define MyAppExeName "chatterino.exe"
+
+; used in build-installer.ps1
+; if set, must end in a backslash
+#ifndef WORKING_DIR
+#define WORKING_DIR ""
+#endif
+
+[Setup]
+; NOTE: The value of AppId uniquely identifies this application. Do not use the same AppId value in installers for other applications.
+; (To generate a new GUID, click Tools | Generate GUID inside the IDE.)
+AppId={{F5FE6614-04D4-4D32-8600-0ABA0AC113A4}
+AppName={#MyAppName}
+AppVersion={#MyAppVersion}
+VersionInfoVersion={#MyAppVersion}
+;AppVerName={#MyAppName} {#MyAppVersion}
```

Comment:
Shouldn't this be `AppVerName={#MyAppName} {#MyAppVersion}` (i.e. include the version)?

Reply:
in the old settings, the version (if everything is set to default) is shown in the versions column
in the new settings, you must click once on the entry to display the version.
It could be in the name too, but is that necessary?

Reply:
[The documentation](https://jrsoftware.org/ishelp/index.php?topic=setup_versioninfoversion) says:

> **Default value**: `AppName` version `AppVersion`
> ...
> This directive specifies the name of the application plus its version number.

So I think it's fine to have it as the default value.

Reply:
<code>AppVerName={#MyAppName} {#MyAppVersion}</code>  should be fine